bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json

```diff
@@ -1,173 +1,173 @@
-    "b01f78cb-334d-49f9-b2c4-f456589bbf6f" : "新しいレポート",
-    "3838ba75-facb-4d62-b237-c3c491205a4e" : "Expense Report Details",
-    "163a4c2a-42b4-4e21-99f3-b898b09cbfb8" : "新しいレポート",
-    "05f29eae-149e-4ee8-90cc-4fbf64d7cc75" : "経費ポリシー",
-    "4facd8ae-fe98-439d-b704-a75a8cb01165" : "承認",
-    "db968206-ca39-4665-968c-c1200780dc4c" : "自分の保留中の承認",
-    "86ce9dbd-5425-4305-a3e9-08e4a4768808" : "レポート名",
-    "dfdd1224-36e3-477e-8de7-91dcd3586357" : "金額",
-    "c8a202f6-2903-48fc-ba88-3cdeec91ba5a" : "表示するデータがありません",
-    "dde0da2b-297f-4bd5-9252-6d909bd86777" : "総額",
-    "23e27e35-0cd2-4ba8-8135-7be74e68f707" : "新しい申請 (変更)",
-    "eef7a19b-7e12-4926-aee3-12024a512461" : "承認リマインダー",
-    "e86f1bad-32a5-4eff-b779-fe9116481dc3" : "承認",
-    "9e142393-5888-4bb5-b456-fb249d268a5b" : "レポートの確認",
-    "ceddd6ca-18a0-477f-bd4d-7ef289f3d142" : "支出カテゴリ",
-    "e437145f-7c5e-4a59-8c2c-2d1d1217a998" : "Expenses",
-    "2744886b-e69e-458b-84e7-aca2d418dbb2" : "Vendor",
-    "45244d60-0585-403b-86e6-5fd2d4365004" : "所有者",
-    "f9378123-5afe-4f81-89c1-d3c6d7716a48" : "承認が見つかりませんでした。",
-    "6ae21628-5879-4698-a3d1-f38ebceed980" : "レポートの検索",
-    "962c4992-348d-4859-a132-49490d2d4a1a" : "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
-    "f4490947-4501-4d48-a3c6-9b9b2185d515" : "状態",
-    "80f5823a-c022-4c92-899a-473409fdde90" : "{{format total \"decimal\"}} {{currency_code}}",
-    "f5fd20ba-44af-4a04-8c9b-3d5b620561d9" : "レポート承認リマインダー",
-    "e408abc3-b922-4ee1-b073-5f07927b9494" : "日付",
-    "6dd5904d-632b-4dbe-8e34-6204371a9aeb" : "保留中のレポートを承認するか、差し戻します。 ",
-    "e1dfed7f-d2be-4265-8a3a-ff81aa2d296a" : "提出日",
-    "675acaee-959c-42f2-9640-d02dc2a9f0b2" : "{{owner_name}} が \"{{name}}\" の経費レポート ({{format total \"decimal\"}} {{currency_code}}) を提出しました。",
-    "7eed4e88-40a2-448b-b889-caaa7d43c1c1" : "金額",
-    "62a1ef1a-e1b2-446e-9ee6-7f4077efab8f" : "{{owner_name}} が \"{{name}}\" の経費レポート ({{format total \"decimal\"}} {{currency_code}}) を提出しました。",
-    "fedc17e1-fa84-44e4-8aec-c19fee060d73" : "ベンダーの説明",
-    "19a7223a-609b-4484-9c05-02308ee35c8e" : "Total",
-    "89d9cc47-a75d-4147-8bcf-6a0bb15e501a" : "Cash Advance",
-    "480ee02e-15a8-4b74-ae12-b7093633a8c6" : "提出者"
+  "99cfe590-6f49-4342-925c-c8c5e902989e": {
+    "440e9042-6cce-49f0-a88d-e57095d71020": "{{format total \"decimal\"}} {{currency_code}}",
+    "2e6177a8-7eca-426e-a07f-559d1d227de0": "経費の詳細",
+    "72fbc197-667b-4303-83d6-3cabce38ca50": "コメント",
+    "067268a4-44f9-4d01-b0f6-7b8e0d3d2d86": "タイプ",
+    "5911041b-1152-44ae-a882-1e05f8f303fb": "{{owner_name}} submitted a {{format total \"decimal\"}} {{currency_code}} expense report for \"{{name}}\".",
+    "63520869-6193-4192-8e83-c6025d5cee76": "自分の保留中の承認",
+    "d2d0d50b-1ef0-4c7a-aa2e-5b98493f473f": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "46de609b-f5d0-461b-952a-08c06402a010": "差し戻す",
+    "0505afb5-3153-4d56-b5d8-bc7864d21baf": "Expense Details",
```

Comment:
Why not translate this string?

Reply:
@yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.